src/components/Categories/ProductCard.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  Button,
+  // Button,
+  Card,
+  // CardActions,
+  CardContent,
+  CardMedia,
+  Stack,
+  Typography,
+} from "@mui/material";
+
+export const ProductCard = () => {
```

Comment:
It should have the information about the product coming from the props. 
when you connect to the backend, this ProductCard will not do any requests to get the product information. It will come from the parent component

Reply:
done